Cargo.toml

```diff
@@ -158,6 +158,9 @@ criterion = "0.6"
 serde = { version = "1.0.106", features = ["derive"] }
 tokio = { version = "1.0.1", features = ["full", "test-util"] }
 hyper = { version = "1.1", features = ["server"] }
+indexmap = { version = "2.10.0", features = ["serde"] }
+linked-hash-map = { version = "0.5.6", features = ["serde_impl"] }
+fxhash = { version = "0.2.1" }
```

Comment:
Are any of these crates used? They're not showing up when I Ctrl-F.

Having both `indexmap` and `linked-hash-map` is kind of redundant since they both do the same thing. I tend to prefer `indexmap` since it's more actively maintained and contains less `unsafe` code.